recipes/caiman/LICENSE.txt

```diff
@@ -0,0 +1,339 @@
+GNU GENERAL PUBLIC LICENSE
```

Comment:
Remove this file as conda-build will find this in the source tarball.

Reply:
👍